cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -25,10 +25,10 @@ spec:
           # keep request = limit to keep this container in guaranteed class
```

Comment:
Well, it depends on what is desired: either we fix the comment or the limits to restore the truth.

I believe the latter is the best option to get the Guaranteed QoS on that pod:

> If limits and optionally requests (not equal to 0) are set for all resources across all containers and they are equal, then the pod is classified as Guaranteed.

https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-qos.md#qos-classes